modules/jfx.incubator.input/src/main/java/jfx/incubator/scene/control/input/InputMap.java

```diff
@@ -360,7 +368,7 @@ private Set<KeyBinding> collectKeyBindings(FunctionTag tag) {
      * @param tag the function tag
      */
     // TODO this should not affect the skin input map, but perhaps place NULL for each found KeyBinding
-    public void unbind(FunctionTag tag) {
+    public void unregister(FunctionTag tag) {
```

Comment:
For symmetry (and better clarity), I still think the name should have `KeyBindingsFor` in the name, perhaps `removeKeyBindingsFor` or `unregisgterKeyBindingsFor`.